src/cap-description.adoc

```diff
@@ -349,6 +349,12 @@ Decoding the bounds:
 top:    t = { a[XLENMAX - 1:E + MW] + ct, T[MW - 1:0]    , {E{1'b0}} }
 base:   b = { a[XLENMAX - 1:E + MW] + cb, B[MW - 1:0]    , {E{1'b0}} }
 ```
+NOTE: The representable range
+ is defined by the space beneath the bottom address bit (`a[E + MW]`)
+ in the expression above. If the capability address changes causing that address bit
+ to change, then the representable range is violated and so the tag will be cleared
+ by an instruction such as <<CINCOFFSET>>. This is represented by a range
+ of `s=2^E+MW^` in xref:cap_bounds_map[xrefstyle=short].
```

Comment:
@tariqkurd-repo : Isn't this similar to the description [here](https://github.com/riscv/riscv-cheri/pull/22/files#diff-2ac200119dffa8d1c8c860fa8f9d6614dc07cf4563a3cc1820cceb25152a12fbL497)?
```
NOTE: The encoding of the bounds depends upon the leading 1 of the address
which is used to determine the exponent. If the leading 1 of the address moves
then the bounds will need to be recalculated. Instructions like <<CINCOFFSET>>
and <<CSETADDR>> update the address field but do not recalculate the bounds.
Therefore, if the leading 1 moves relative to when the bounds were calculated
then the tag is cleared on the result as the encoding has been invalidated.
```

Reply:
yes - but actually that description is inaccurate which was my misunderstanding - the new description is accurate. I'll resolve the differences and push more commits.